apps/condo/domains/banking/schema/BankAccount.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * Generated by `createschema banking.BankAccount 'organization:Relationship:Organization:CASCADE; tin:Text; country:Text; routingNumber:Text; number:Text; currency:Text; approvedAt?:DateTimeUtc; approvedBy?:Text; importId?:Text; territoryCode?:Text; bankName?:Text; meta?:Json; tinMeta?:Json; routingNumberMeta?:Json'`
+ */
+
+const { get } = require('lodash')
+
+const { Text, DateTimeUtc, Select } = require('@keystonejs/fields')
+const { Json } = require('@condo/keystone/fields')
+const { GQLListSchema } = require('@condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@condo/keystone/plugins')
+
+const access = require('@condo/domains/banking/access/BankAccount')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+const { IMPORT_ID_FIELD, CURRENCY_CODE_FIELD } = require('@condo/domains/common/schema/fields')
+const { COUNTRIES } = require('@condo/domains/common/constants/countries')
+
+const { validateTin } = require('@condo/domains/banking/utils/validate/tin.utils')
+const { validateRoutingNumber } = require('@condo/domains/banking/utils/validate/routingNumber.utils')
+const { validateNumber } = require('@condo/domains/banking/utils/validate/number.utils')
+
+
+const BankAccount = new GQLListSchema('BankAccount', {
+    schemaDoc: 'Bank account, that will have transactions, pulled from various integrated data sources. Transactions will be marked by categories of incomes and expenses.',
+    fields: {
+        organization: ORGANIZATION_OWNED_FIELD,
+
+        tin: {
+            schemaDoc: 'Tax Identification Number',
+            type: Text,
+            isRequired: true,
+            hooks: {
+                validateInput: ({ existingItem, resolvedData, addFieldValidationError }) => {
+                    const newItem = { ...existingItem, ...resolvedData }
+
+                    const country = get(newItem, 'country')
+                    const tin = get(newItem, 'tin')
+
+                    const { result, errors } = validateTin(tin, country)
+
+                    if ( !result ) {
+                        addFieldValidationError(errors[0])
+                    }
+                },
+            },
+        },
+
+        country: {
```

Comment:
we can probably use it from the organization. or do you think that we can have two accounts from different countries?

Reply:
This question was addressed in latter comment

> can we use it from the organization?
> I thought we have discussed that sometimes you (as an organization) need to create bankAccounts for separate organizations. I think the case was connected with payments to some partner organizations ... or resource providers